Ada Examples/Examples Based On OpenGL Superbible/Simplest/src/run.ads

```diff
@@ -0,0 +1,9 @@
+
+with GL.Objects.Programs;
+
+with Window_Types;
+
+package Run is
+    procedure Main_Loop(Main_Window    : in out Window_Types.tWindow;
+                                         Rendering_Program   : GL.Objects.Programs.Program);
```

Comment:
See my restructuring comments with respect to render.
Run now includes Starup (which I originally meant to change to Setup), Render_Dot and Main_Loop.
This is my best effort (so far) at providing an example-specific package.